common/jmeter/perf-test-is.sh

```diff
@@ -466,6 +467,23 @@ function run_tenant_test_data_scripts() {
     done
 }
 
+function run_api_authorization_test_data_scripts() {
```

Comment:
do we need this function?
We have a dedicated function to run the test data script.
Additionally, if you added this since you introduce a new param, let's pass that as well, if it is considered only in the script that'll work, otherwise it'll ignored I guess. Verify it and remove this function.